RocketChat/Rocket.Chat.ReactNative
# [NEW] Redesign `ReactionsList` and migrate to bottom sheet

## Proposed changes

* Redesigned `ReactionsList` component which was previously rendered as `Reactions Modal` and migrate it to bottom sheet.

## Issue(s)	


## How to test or reproduce

* Long press on reactions below a message and see if the reactions list work as expected.

## Screenshots

|Before|After|
|-------|------|
|![reactions-list-before](https://user-images.githubusercontent.com/77742477/177613809-62e2b9e5-f312-43f6-9f22-e624692aa696.png)|![new reactions-list](https://user-images.githubusercontent.com/77742477/177613827-161ffe70-2932-4353-b146-013319b2dd9d.png)|


## Types of changes




## Checklist



## Further comments
